api.py

```diff
@@ -39,7 +39,7 @@ def to_pretty_json(value):
 def dinghy_html(req, resp):
     """Index route to Dinghy-ping input html form"""
     resp.content = api.template(
-        'ping_input.html',
+        'dinghy_index.html',
```

Comment:
I prefer to have index.html for the name and drop any dinghy prefix this will simplify the naming also for our routes we can remove that /dinghy

Reply:
For sure let's do that. I'm going to merge this for now so we can deploy a beta live version on the stage cluster. Feel free to open a new PR